beta/project-griffon/using-project-griffon/validation-summary-and-project-griffon.md

```diff
@@ -0,0 +1,69 @@
+# Validation Summary
+
+## Overview
+
+You may use the Validation Summary feature to quickly and easily manage JavaScript functions to validate events in a Project Griffon session.
+
+## Getting Started
+
+To use this view, complete the following steps:
+
+1. [Setup Project Griffon](../set-up-project-griffon.md)
+2. [Create](./#creating-sessions) and [connect](./#connecting-to-a-session) to a Project Griffon session
+3. In the **Home** view, select **Validation Summary**
+
+![Validation Summary](../../../.gitbook/assets/screen-shot-2021-02-08-at-2.49.28-pm.png)
+
+## Validation Management
+
+When the view is loaded, it provides a panel on the left that contains a categorized list of validation results and a panel on the right to create and edit JavaScript validation functions.
+
+## Writing a Validation Function
+
+This feature allows you to create, edit, or delete validation functions for your Project Griffon sessions. 
+
+1. Select **Create a New Validation**.
+2. Enter a **name** to identify the validation, then provide a **category** and a **description**.
+3. Edit the code in the editor to validate the events for your Project Griffon session.
+4. Click **Test** to preview the results of the function in the left panel.
+
+After you have tested your function, you can click **Upload** to save your validation. 
+
+### Event Definition
+
+| Key | Type | Description |
+| :--- | :--- | :--- |
+| `uuid` | String | Universally unique identifier for the event. |
+| `timestamp` | Number | Timestamp from the device when the event was sent from the SDK. |
```

Comment:
This seems specific to Mobile SDK. Let's make it more generic so it applies to web/server also.